components/navbar/navbar.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="./navbar.css" rel="stylesheet" />
+    <link rel="stylesheet" href="/components/avatar/avatar.css" />
```

Comment:
It looks like navbar is dependant upon avatar for inheriting some styles. In that case, you can use the @import tag in your navbar.css file and import the avatar.css there.